frontend/public/components/_persistent-volume-claim.scss

```diff
@@ -0,0 +1,4 @@
+.co-pvc-donut {
+  max-width: 325px;
```

Comment:
In the deployments page, we are not dedicating an entire column to it. And there is a restriction of height to grow there. But here since an entire column is dedicated, IMO 325px is okayish. I tried it with 250px it doesn't look bad either. 
Scrshot with 170px:
![Screenshot from 2020-06-26 15-15-58](https://user-images.githubusercontent.com/54092533/85844305-21fc2d00-b7c0-11ea-943f-f2fe6094afd1.png)

Reply:
Hmm, yeah, For consistency, I'm thinking that maybe that donut should be above the details instead of on the right, and should be the same size as other donuts in console. I believe every other detail view with charts shows them on top.

cc @openshift/team-ux-review @spadgett

Reply:
Talked with @yuvalgalanti and he approved with this move. Making it the std. size that is used by PodRing in deployments.